tests/conftest.py

```diff
@@ -52,8 +52,26 @@ def cart(db):  # pylint: disable=W0613
 
 
 @pytest.fixture
-def customer_user(db):  # pylint: disable=W0613
-    return User.objects.create_user('test@example.com', 'password')
+def default_billing_address(db):  # pylint: disable=W0613
```

Comment:
This might be used as any address, not exactly default billing, for the purpose of creating `customer_user`, and as I'm aware - it is.
I think that `address` is a better fit, as and advantage we won't end up with several:
`address = default_billing_address` in tests, to fit everything within 80 chars